bundle/jsky.util/src/main/java/jsky/util/DateUtil.java

```diff
@@ -17,12 +17,12 @@
 public class DateUtil {
 
     /**
-     * Return a string with the UTC time in the format /mm/dd/yy hh:mm:ss,
+     * Return a string with the UTC time in the format yyyy-MMM-dd hh:mm:ss,
      * given the current UTC time in ms.
      */
     public static String formatUTC(long time) {
```

Comment:
No usages found with IDEA in ocs. There is also another formatting method below that didn't change but also couldn't find any usages. ¿Do you think those methods are used by ocs API consumers? ¿Can we remove them?